src/components/AcceptDialog.tsx

```diff
@@ -22,6 +26,50 @@ const initialValues = {
 };
 
 const AcceptDialog: FunctionComponent<Props> = (props) => {
+  const [submitError, setSubmitError] = useState(false);
+  const [errorMessage, setErrorMessage] = useState<string>();
+  const handleAccept = useCallback(
```

Comment:
Small one - a new line between `handleAccept` and the states would help with readability

Reply:
Addressed